individual_trajectories_bridge/src/FollowJointTrajectoryAction.cpp

```diff
@@ -11,7 +11,7 @@ FollowJointTrajectoryAction::FollowJointTrajectoryAction(std::string name, ros::
   sub_action_cancel_ = nh.subscribe( action_name_ + std::string("/cancel"), 1, &FollowJointTrajectoryAction::actionCancelCallback, this);
 }
 
-void FollowJointTrajectoryAction::actionCancelCallback(const actionlib_msgs::GoalID::ConstPtr& msg)
+void FollowJointTrajectoryAction::actionCancelCallback(const actionlib_msgs::GoalID::ConstPtr&)
```

Comment:
Did you test if only this fix is enough without modifying the CMake?

Reply:
This is only to hide a warning during compilation. Cmake was enough to fix the issue yes.

Reply:
Could you please create a new PR solving only the warning?

Reply:
Yep that is a good catch. Lets make a PR to fix this warning.